Update/wata_007.txt

```diff
@@ -318,7 +318,7 @@ void main()
 		   NULL, " There's a lot of building tents and setting up chairs.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 2, "ps3/s19/02/500200918", 128, TRUE);
 	OutputLine(NULL, "結構重労働って聞いてるよ。」",
-		   NULL, " I hear it's pretty hard physical labor!\"", GetGlobalFlag(GLinemodeSp));
+		   NULL, " I heard it was pretty hard physical labor!\"", GetGlobalFlag(GLinemodeSp));
```

Comment:
This is incorrect - she's describing what she hears about the act of doing this work in general, which is present tense.